# a phase space marker

# LED id
int64 id

# 3D coordinates of the LED
geometry_msgs/Point point